operator/internal/manifests/rules_config.go

```diff
@@ -0,0 +1,58 @@
+package manifests
+
+import (
+	"fmt"
+
+	"github.com/grafana/loki/operator/internal/manifests/internal/rules"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// RulesConfigMap returns a ConfigMap resource that contains
+// all loki alerting and recording rules as YAML data.
+func RulesConfigMap(opts *Options) (*corev1.ConfigMap, error) {
```

Comment:
Would it make sense to move this code to `github.com/grafana/loki/operator/internal/manifests/internal/rules` as well?

Reply:
No this belongs in the `manifests` package as it is the actual resource builder. We generally move only helpers and templates into the internal package.